<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exercice1Class</class>
 <widget class="QMainWindow" name="exercice1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>402</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>exercice1</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">exercice1Class{
	background-color: rgb(97, 121, 255);
}</string>
   </property>
   <widget class="QLabel" name="labelScore">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonLancerDe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>lancer le dé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAfficheScoreDe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>afficher le score du dé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonResetScore">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>reset mon score</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>score :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDe">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>historique:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listhistorique">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelmenuprincipal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>choisir le mode de Dé</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelbienvenu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>bienvenu sur le symulateur de lanceur de De !! </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNombreLancer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de lancer :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNlancer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonNlancer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>lance le simulateur</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResultat">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)
</string>
    </property>
    <property name="text">
     <string>Résultat</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelgif">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDeGif">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufichier">
    <property name="title">
     <string>fichier</string>
    </property>
    <addaction name="actionquitte"/>
   </widget>
   <widget class="QMenu" name="menuchoisir">
    <property name="title">
     <string>choisir</string>
    </property>
    <addaction name="actionde"/>
    <addaction name="actiondehisto"/>
   </widget>
   <addaction name="menufichier"/>
   <addaction name="menuchoisir"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionde">
   <property name="text">
    <string>de</string>
   </property>
  </action>
  <action name="actiondehisto">
   <property name="text">
    <string>dehisto</string>
   </property>
  </action>
  <action name="actionquitte">
   <property name="text">
    <string>quitte</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="exercice1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonLancerDe</sender>
   <signal>clicked()</signal>
   <receiver>exercice1Class</receiver>
   <slot>lanceDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonAfficheScoreDe</sender>
   <signal>clicked()</signal>
   <receiver>exercice1Class</receiver>
   <slot>afficheDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonResetScore</sender>
   <signal>clicked()</signal>
   <receiver>exercice1Class</receiver>
   <slot>resetScore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionde</sender>
   <signal>triggered()</signal>
   <receiver>exercice1Class</receiver>
   <slot>AffichedeNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actiondehisto</sender>
   <signal>triggered()</signal>
   <receiver>exercice1Class</receiver>
   <slot>AfficheDeHisto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonNlancer</sender>
   <signal>clicked()</signal>
   <receiver>exercice1Class</receiver>
   <slot>Nlancer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionquitte</sender>
   <signal>triggered()</signal>
   <receiver>exercice1Class</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>afficheScore()</slot>
  <slot>afficheDe()</slot>
  <slot>resetScore()</slot>
  <slot>lanceDe()</slot>
  <slot>AffichedeNormal()</slot>
  <slot>AfficheDeHisto()</slot>
  <slot>Nlancer()</slot>
  <slot>exit()</slot>
 </slots>
</ui>
